pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -129,6 +129,7 @@ def get_unit_data(self, unit):
 
 class Command(BaseCommand):
     help = "Load Translation Memory with translations"
+    args = "[files]"
```

Comment:
I think that `[files]` clearly states that files are optional, so this stands for both file and non-file versions.

Displaying two different usage snippets (both at the same time) would require to override the `usage()` method, but I don't think it is actually necessary.